### PR TITLE
Fix validators query

### LINKED ISSUE #226: Validator query returns only 10 elements
On testnet-3 `tgrade q poe validators`
* Support `--limit `

### PR DESCRIPTION
**Pull request by alpe**
Resolves #226

### CODE REVIEW

**Review by maurolacy — APPROVED** [review]
Looks good.

I guess this issue was caused by the new pagination functionality / defaults. And that I never ended up enabling those setting in the cli, right?

File: x/poe/client/cli/query.go [hunk 1]
```diff
@@ -124,10 +124,9 @@ $ %s query poe validators
 			if err != nil {
 				return err
 			}
-			_ = pageReq // todo (Alex): support pagination
```
Comment (ethanfrey) [inline x/poe/client/cli/query.go:127]: Nice that the comments said what to fix.

**Review by ethanfrey — APPROVED** [review]
lgtm

**Comment by alpe** [comment]
I have updated the CLI params to use `key`.

**Comment by ethanfrey** [comment]
Looks good to merge

**Pull request merged**